pom.xml

```diff
@@ -333,6 +340,13 @@
                 <scope>test</scope>
             </dependency>
 
+            <dependency>
```

Comment:
Small one, but I think it makes more sense for this dependency to be moved below the core one.

Reply:
moved inline below core